content/react/concepts/state/state.md

```diff
@@ -76,6 +76,41 @@ class Car extends React.Component {
 }
 ```
 
+## Using Property Initializers
+
+With Property Initializers you do not need to set state or bind methods in the constructor:
+
+```js
+class Car extends React.Component {
+  // no need to set state in constructor
+  state = {
+    brand: 'Chevrolet',
+    model: 'Malibu',
+    color: 'white',
+    year: 1998,
+  };
+
+  // no need to bind methods in constructor; use arrow functions to declare methods
+  upgrade = () => {
+    this.setState({ year: this.state.year + 1 });
+  };
```

Comment:
I'm wondering if this part is more relevant to _components_ as opposed to _state_?

Reply:
I think you are correct sir, removed.